specification/batch/data-plane/Microsoft.Batch/stable/2018-03-01.6.1/BatchService.json

```diff
@@ -11546,7 +11546,7 @@
             "$ref": "#/definitions/ResourceFile"
```

Comment:
All places where ResourceFiles surface should get doc updates: job-prep, starttask, job-schedule? job-release?  Ask Matt-C for the whole list.